src/tests/catwalk_tests/test_baselines.py

```diff
@@ -88,6 +90,15 @@ def test__convert_string_rule_to_dict(self):
         expected_results = {"feature_name": "x1", "operator": "gt", "threshold": 0}
         assert results == expected_results
 
+    def test_validate_operator(TestCase):
```

Comment:
You definitely don't want to name your method's reference to the `TestCase` *instance* – (usually `self`) – `TestCase`.

Also, however, it doesn't look like this test requires any of the test class's set-up. If you wanted to just make it a plain test function, at the module level, you could. I see that you're testing something to do with thresholds, so perhaps you want to gather all those under this namespace; however, you're not testing the `SimpleThresholder`.

So, you _could_, instead, define a module-level test function:

```
@pytest.mark.parametrize('operator', [">", ">=", "<", "<=", "=="])
def test_validate_operator(operator):
    series = pd.Series([1, 2, 3, 4, 5])
    pd_operator = validate_operator(operator, "This is a  dummy rule.")
    result = getattr(series, pd_operator)(series)
    assert ptypes.is_bool_dtype(result)
```